app/web_ui/src/routes/(app)/settings/manage_tools/[project_id]/add_tools/+page.svelte

```diff
@@ -1,183 +1,145 @@
-  import { KilnError, createKilnError } from "$lib/utils/error_handlers"
-
-  // Form fields
-  let name = ""
-  let server_url = ""
-  let description = ""
-
-  // Headers as array of key/value pairs
-  interface HeaderPair {
-    key: string
-    value: string
+  import AppPage from "../../../../app_page.svelte"
+  import { ui_state } from "$lib/stores"
+
+  // Type definition for MCP tool items
+  interface McpServer {
+    name: string
+    description: string
+    server_url: string
+    headers: { key: string; value: string; placeholder: string | null }[]
+    button_text: string
   }
 
-  let headers: HeaderPair[] = []
-
-  // Form state
-  let error: KilnError | null = null
-  let submitting = false
-
-  function buildHeadersObject(): Record<string, string> {
-    const headersObj: Record<string, string> = {}
-
-    for (const header of headers) {
-      if (header.key.trim() && header.value.trim()) {
-        headersObj[header.key.trim()] = header.value.trim()
-      }
-    }
-
-    return headersObj
+  // Helper function to navigate to remote MCP page with pre-filled data
+  function connectRemoteMcp(item: McpServer) {
+    goto(
+      `/settings/manage_tools/${$ui_state?.current_project_id}/add_tools/remote_mcp`,
+      {
+        state: {
+          name: item.name,
+          description: item.description,
+          server_url: item.server_url,
+          headers: item.headers,
+        },
+      },
+    )
   }
 
-  async function connect_remote_mcp() {
-    try {
-      error = null
-      submitting = true
-
-      // Validate required fields
-      if (!name.trim()) {
-        throw new Error("Name is required")
-      }
-      if (!server_url.trim()) {
-        throw new Error("Server URL is required")
-      }
-      if (headers.length > 0) {
-        for (const header of headers) {
-          if (!header.key.trim()) {
-            throw new Error("Header name is required")
-          }
-          if (!header.value.trim()) {
-            throw new Error("Header value is required")
-          }
-        }
-      }
-
-      const headersObj = buildHeadersObject()
-
-      const { data, error: api_error } = await client.POST(
-        "/api/projects/{project_id}/connect_remote_mcp",
+  const sampleMcpServers: McpServer[] = [
+    {
+      name: "GitHub",
+      description:
+        "Adds ability to read repositories and code files, manage issues and PRs, analyze code, and automate workflow.",
+      server_url: "https://api.githubcopilot.com/mcp/",
+      headers: [
         {
-          params: {
-            path: {
-              project_id: $page.params.project_id,
-            },
-          },
-          body: {
-            name: name.trim(),
-            server_url: server_url.trim(),
-            headers: headersObj,
-            description: description.trim() || null,
+          key: "Authorization",
+          value: "",
+          placeholder: "Bearer {GITHUB_PERSONAL_ACCESS_TOKEN}",
```

Comment:
Maybe set value to "Bearer REPLACE_WITH_GITHUB_PERSONAL_ACCESS_TOKEN"

That way they don't have to type "Bearer "

P2.